src/rgw/rgw_common.h

```diff
@@ -524,6 +524,21 @@ enum http_op {
   OP_UNKNOWN,
 };
 
+inline static std::string http_op_to_str(http_op op) {
+  static std::vector<std::string> htt_ops_str = {
+      {"OP_GET"},
```

Comment:
from: https://docs.aws.amazon.com/AmazonS3/latest/userguide/LogFormat.html#log-record-fields
it show that the format the http method is: `PUT`, `GET`, `HEAD` etc. without the `OP_`

also, see [here](https://github.com/ceph/ceph/pull/17754/files#diff-fa125017f7100671583ccc8aea06e4167db43b718ebc32dce81663799a0ea19bR226)